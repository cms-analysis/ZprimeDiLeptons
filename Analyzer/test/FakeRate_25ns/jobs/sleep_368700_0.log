000 (368700.000.000) 09/05 19:57:17 Job submitted from host: <90.147.169.224:9618?addrs=90.147.169.224-9618&noUDP&sock=2441165_0900_3>
...
001 (368700.000.000) 09/05 20:18:28 Job executing on host: <90.147.168.135:35244>
...
006 (368700.000.000) 09/05 20:18:37 Image size of job updated: 130688
	88  -  MemoryUsage of job (MB)
	89172  -  ResidentSetSize of job (KB)
...
006 (368700.000.000) 09/05 20:18:42 Image size of job updated: 183260
	88  -  MemoryUsage of job (MB)
	89172  -  ResidentSetSize of job (KB)
...
005 (368700.000.000) 09/05 20:18:43 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:05, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:05, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	98524  -  Run Bytes Sent By Job
	69443  -  Run Bytes Received By Job
	98524  -  Total Bytes Sent By Job
	69443  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :      214       75   2663893
	   Memory (MB)          :       88     2500      2500
...
