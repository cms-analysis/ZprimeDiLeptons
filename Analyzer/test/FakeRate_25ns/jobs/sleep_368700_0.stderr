mv: impossibile fare stat di `/home/condor/dir_1569/CMSSW803-Analyse_ZprimeToMuMu_13TeV.root': No such file or directory
mv: impossibile fare stat di `/home/condor/dir_1569/CMSSW803-Analyse_ZprimeToMuMu_13TeV_cand.txt': No such file or directory
